Windows Build Documentation Missing .NET 3.5 Requirement

### What You Are Seeing?

Microsoft .NET Framework 3.5 (net-3.5) is not installed, or not correctly configured.

### What is Expected?

I expect the build to pass.

### How Did You Get This To Happen? (Steps to Reproduce)

1. Vagrant up a new windows server 2016 machine
2. install git
3. clone this repository
4. run .\build.bat 

### Output Log

```
     [echo] Removing and adding C:\Users\vagrant\source\repos\choco\build_output.
     [echo] Build for chocolatey started at 10/02/2018 13:04:17 with UppercuT.
     [echo] Using UppercuT version: 1.4.2.0.
     [echo] ====================
     [echo] UppercuTBuild
     [echo] ====================
                 [echo] ====================
                 [echo] GitVersioner
                 [echo] ====================
                 [echo] Getting revision number from working directory 'C:\Users\vagrant\source\repos\choco' or remotely at https://github.com/chocolatey/choco.
     [echo] Using revision number: 0.10.11-49-gbfe351b7 and hash: 0.10.11-49-gbfe351b7.
     [echo] Using build number 0.
     [echo] Version full updated to 0.10.11.0.
     [echo] Compiled code drop directory will be C:\Users\vagrant\source\repos\choco\code_drop.
                 [echo] ====================
                 [echo] VersionBuilder
                 [echo] ====================
                 [echo] Generating a strong name key (C:\Users\vagrant\source\repos\choco\chocolatey.snk) using C:\Program Files (x86)\Microsoft SDKs\Windows\v6.0A\Bin\sn.exe if the key file doesn't already exist.
                 [echo] Generating SolutionVersion.cs with assembly version 0.10.11.0.
                 [echo] ====================
                 [echo] BuildInformation
                 [echo] ====================
                 [echo] Generating XML file with some information to possibly get merged in at C:\Users\vagrant\source\repos\choco\build_output\_BuildInfo.xml.
            
            BUILD FAILED
            
            Microsoft .NET Framework 3.5 (net-3.5) is not installed, or not correctly configured.
                Framework directory 'C:\Windows\Microsoft.NET\Framework64\v3.5' does not exist.
            


Nested build failed.  Refer to build log for exact reason.
```

Hey @ferventcoder ,

While i've never used Nant, it looks like it or something it uses has a requirement on .NET Framework v3.5

```
             [property] Property "run.nuget" already exists, and "overwrite" is set to false.
             [property] Property "app.ruby" already exists, and "overwrite" is set to false.
             [property] Property "app.powershell" already exists, and "overwrite" is set to false.

            BUILD FAILED

            Microsoft .NET Framework 3.5 (net-3.5) is not installed, or not correctly configured.:
            NAnt.Core.BuildException: Microsoft .NET Framework 3.5 (net-3.5) is not installed, or not correctly configured. ---> System.ArgumentException: Framework directory 'C:\Windows\Microsoft.NET\Framework64\v3.5' d
oes not exist.
               at NAnt.Core.FrameworkInfo.get_FrameworkDirectory()
               at NAnt.Core.FrameworkInfo.Validate()
               --- End of inner exception stack trace ---
               at NAnt.Core.FrameworkInfo.Validate()
               at NAnt.Core.Project.set_TargetFramework(FrameworkInfo value)
               at NAnt.Core.Tasks.PropertyTask.ExecuteTask()
               at NAnt.Core.Task.Execute()
               at NAnt.Core.Project.InitializeProjectDocument(XmlDocument doc)
               at NAnt.Core.Project.Execute()
               at NAnt.Core.Project.Run()

            Total time: 0.3 seconds.


BUILD FAILED - 0 non-fatal error(s), 10 warning(s)

Nested build failed.  Refer to build log for exact reason.:
NAnt.Core.BuildException: Nested build failed.  Refer to build log for exact reason.
   at NAnt.Core.Tasks.NAntTask.RunBuild(FileInfo buildFile)
   at NAnt.Core.Tasks.NAntTask.ExecuteTask()
   at NAnt.Core.Task.Execute()
   at NAnt.Core.Target.Execute()
   at NAnt.Core.Project.Execute(String targetName, Boolean forceDependencies)
   at NAnt.Core.Task.Execute()
   at NAnt.Core.Target.Execute()
   at NAnt.Core.Project.Execute(String targetName, Boolean forceDependencies)
   at NAnt.Core.Project.Execute()
   at NAnt.Core.Project.Run()

Total time: 1.7 seconds.
```

--- a/README.md
+++ b/README.md
@@ -124,6 +124,7 @@ There is a `build.bat`/`build.sh` file that creates a necessary generated file n
 #### Windows
 Prerequisites:
 
+ * .NET Framework 3.5 (This is a windows feature installation).
  * .NET Framework 4+
  * Visual Studio is helpful for working on source.
  * ReSharper is immensely helpful (and there is a `.sln.DotSettings` file to help with code conventions).

